### ISSUE TITLE
Accessibility: Buttons must have discernible text

### ISSUE THREAD

**Issue by chinthas-mh** [6pac/SlickGrid#586]
Ran accessibility tool AXE chrome plug in and it has reported below critical issue
Slickgrid version AE 2.26

Element location
.slick-gridmenu-button

Element source
<button class="slick-gridmenu-button fa fa-bars"></button>

Fix at least one (1) of these issues:
•	Element does not have inner text that is visible to screen readers
•	aria-label attribute does not exist or is empty
•	aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
•	Element's default semantics were not overridden with role="none" or role="presentation"
•	Element has no title attribute

**Comment by ghiscoding** [comment]
So we are patiently waiting for your PR (Pull Request) as contribution to the project 😉

**Comment by ghiscoding** [comment]
closed by #764 to be released in next major v4.0
